integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java

```diff
@@ -558,6 +615,32 @@ public int rmdir(String path) {
   }
 
   /**
+   * Changes the size of a file. This operation would not succeed because of Alluxio's write-once
+   * model.
+   */
+  @Override
+  public int truncate(String path, long size) {
```

Comment:
would it make sense to always return an error saying that we don't support `truncate`?

Reply:
No, shell's error message is based on the error code we return. The error message will print to `fuse.log`. So unless the user checks the log, he won't know it.